src/pages/Causes/SignupForm.vue

```diff
@@ -167,7 +167,13 @@ export default {
 		completeCausesBraintree() {
 			// disable form inputs while routing to thanks
 			this.success = true;
+			// fire events
 			this.$kvTrackEvent('Registration', 'successful-causes-reg', 'register-causes');
+			// Track facebook event
+			if (typeof window !== 'undefined' && typeof fbq === 'function') {
+				window.fbq('track', 'CausesSignUp');
```

Comment:
Verified with product, see ticket. Let's go with this so it matches the event for MG sign ups